app.js

```diff
@@ -0,0 +1,15 @@
+const db = require("./db/connection");
+const express = require("express");
+const app = express();
+
+const { errorHandler500 } = require("./controllers/error-handling-controllers");
+const { getCategories } = require("./controllers/categories-controllers");
+
+app.use(express.json());
+app.use((req, res, next) => next());
```

Comment:
This is not doing anything, its just moving straight to your 500 handler, without the error, just remove it